features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/async/RecordedDataQueueHandler.kt

```diff
@@ -99,6 +106,11 @@ internal class RecordedDataQueueHandler {
         val rumContextData = rumContextDataHandler.createRumContextData()
             ?: return null
 
+        // if the view changed then clear the drawable lrucache
+        if (rumContextData.prevRumContext != rumContextData.newRumContext) {
```

Comment:
good call

Reply:
is it needed? given that we convert small icons which can be re-used between the views, maybe we can keep them?

Reply:
I am more worried about the big ones here but that's a good question indeed.